README.md

```diff
@@ -1541,20 +1419,20 @@ We'll use:
             {:name "green" :message "green makes it go faster"}]))
 ```
 
-Now ```chat-messages``` is pointing to the ```atom``` protecting our vector
+Now `chat-messages` is pointing to the `atom` protecting our vector
 of hashes.
 
-Because ```chat-messages``` is pointing to the ```atom```, we can't simply
-```map``` over it in ```generate-message-view```.  Now we have to tell Clojure
-that we want to generate HTML for the content of the atom.  This allows Clojure
-to ensure the messages are always read in a consistent state, even though
-something could be modifiying them.
+Because `chat-messages` is pointing to the `atom`, we can't simply
+`map` over it in `generate-message-view`.  Now, we have to tell
+Clojure that we want to generate HTML for the contents of the atom.  This
+allows Clojure to ensure the messages are always read in a consistent
+state, even though something could be modifiying them.
 
```

Comment:
This might be too much of a nitpick, but atoms just hold a singular value, hence "content" vs. "contents".